pynetbox/models/dcim.py

```diff
@@ -13,12 +13,56 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
+from six.moves.urllib.parse import urlsplit
+
+from pynetbox.core.query import Request
 from pynetbox.core.response import Record, JsonField
 from pynetbox.core.endpoint import RODetailEndpoint
 from pynetbox.models.ipam import IpAddresses
 from pynetbox.models.circuits import Circuits
 
 
+class TraceableRecord(Record):
```

Comment:
I don't love having this here but the trace fn needs to use the dcim objects in the response.. if you have a better way to structure it let me know :)

Reply:
Yeah, that's fair, I don't immediately see any better ways around it if we want to return objects. Will keep thinking about it though.